itests/f3_test.go

```diff
@@ -275,14 +301,23 @@ func setupWithStaticManifest(t *testing.T, manif *manifest.Manifest, testBootstr
 		err = n.NetConnect(ctx, e.ms.PeerInfo())
 		require.NoError(t, err)
 	}
-
 	errgrp.Go(func() error {
 		defer func() {
 			require.NoError(t, manifestServerHost.Close())
 		}()
 		return e.ms.Run(ctx)
 	})
 
+	// Update initial manifest params to shorten the timeouts and backoff for
+	// testing, and assert it is consistently applied to all nodes.
+	e.m.Gpbft.Delta = 250 * time.Millisecond
+	e.m.Gpbft.DeltaBackOffExponent = 1.3
+	e.m.Gpbft.RebroadcastBackoffBase = manif.Gpbft.Delta * 2
+	e.m.Gpbft.RebroadcastBackoffMax = manif.Gpbft.RebroadcastBackoffBase * 2
+	e.m.Gpbft.RebroadcastBackoffExponent = manif.Gpbft.DeltaBackOffExponent
```

Comment:
What about creating a `newTestManifest` and using that instead of `lf3.NewManifest` so we don't have to override these here.

Reply:
Doing it this way has a subtle side effect: it would be relative to whatever the default initial manifest is set to.

Happy to go the other route, though i think we still need to assert it is applied to all nodes on test environment instantiation.

Reply:
There shouldn't be a default initial manifest, right?

Reply:
I.e., the "default initial manifest" is https://github.com/filecoin-project/lotus/blob/f6d4fc7ccb0a53fd49cf01a82ab3a9528345afab/itests/f3_test.go#L212